examples/ota-requestor-app/efr32/BUILD.gn

```diff
@@ -81,6 +82,8 @@ efr32_sdk("sdk") {
 
 efr32_executable("ota_requestor_app") {
   output_name = "chip-efr32-ota-requestor-example.out"
+  include_dirs = [ "include" ]
+  defines = []
```

Comment:
Does moving this include higher change anything in the build sequence? Just curious

Reply:
I don't think it does. Just wanted to have an identical BUILD.gn to the lighting app.